### PR TITLE
Service Location Styling and header level fixes

### PR DESCRIPTION
**Pull request by mpelzsherman**
Styling: https://github.com/department-of-veterans-affairs/va.gov-team/issues/17755
Header levels: https://github.com/department-of-veterans-affairs/va.gov-team/issues/15701

## Testing done
View http://localhost:3001/pittsburgh-health-care/locations/pittsburgh-va-medical-center-university-drive/
Look at Addiction (has a location name) and Cardiology (no location name).

## Acceptance criteria
- [x] styles match spec
- [x] no heading levels are skipped

### CODE REVIEW

File: src/site/facilities/service_location.drupal.liquid [hunk 1]
```diff
@@ -70,11 +83,15 @@
...
+    <div class="vads-u-display--flex vads-u-flex-direction--column" data-template="paragraphs/service_location">
+      {% if haveLocationName %}
+        <h4 class="force-small-header">Phone</h4>
+      {% else %}
+        <h5>Phone</h5>
+      {% endif %}
       {% if locationEntity.fieldPhoneNumbersParagraph %}
         {% for number in locationEntity.fieldPhoneNumbersParagraph %}
-          <a class="vads-u-margin-bottom--1" aria-label="{{ number.entity.fieldPhoneNumber | accessibleNumber }}"
+          <a aria-label="{{ number.entity.fieldPhoneNumber | accessibleNumber }}"
```
Comment (brangi) [inline src/site/facilities/service_location.drupal.liquid:94]: where does `accessibleNumber` come from ? I can't find it assigned above.

Comment (mpelzsherman) [reply src/site/facilities/service_location.drupal.liquid:94]: It's a filter function defined [here](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/filters/liquid.js#L469).

**Review by brangi — APPROVED** [review]

**Review by kelsonic — APPROVED** [review]
LGTM! Couldn't find anything that affects PW! 🙂

File: src/site/layouts/health_care_local_facility.drupal.liquid [hunk 2]
```diff
@@ -1,213 +1,3 @@
-{% comment %}
```
Comment (billfienberg) [inline src/site/layouts/health_care_local_facility.drupal.liquid:1]: Dumb question: Why is it necessary to delete this comment containing the sample JSON content?

**Review by billfienberg — APPROVED** [review]

File: src/site/layouts/health_care_local_facility.drupal.liquid (hunk 2 above)
Comment (mpelzsherman) [reply src/site/layouts/health_care_local_facility.drupal.liquid:1]: The team has agreed that those comments don't provide any value. The actual JSON content is available in the JS console in a much more useful form. The comments can go out of date quickly and can cause confusion.

**Pull request merged**